uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -484,6 +464,22 @@ public function upgrade_1018() {
     return false;
   }
 
+  /**
+   * Disables the Case menu items if Tasks&Assignments is enabled
+   *
+   * @return {boolean}
+   */
+  public function upgrade_1019() {
```

Comment:
why did you moved the logic to 1019 , isn't this 1017 already executed and I see no difference between 1017 and 1019